infra/token-transfer-client/src/constants/index.js

```diff
@@ -1,9 +1,9 @@
-import moment from 'moment'
+import {
+  employeeUnlockDate,
+  investorUnlockDate,
+  lockupBonusRate
+} from '@origin/token-transfer-server/src/shared'
```

Comment:
The client having a dependency on the server will cause it to pull back-end packages such as sequelize. I don't think that's a huge deal. Just pointing this out.

Reply:
I have a [separate file](https://github.com/OriginProtocol/origin/blob/d6fe10694d3067b56c175cd63e51e1afc51f5497/infra/token-transfer-server/src/shared.js) in the server that contains all the shared code.  I have to be very careful with the imports for the reason you mentioned (leave sequelize out, etc). This dependency exists elsewhere in the client for sharing the vesting logic between client and server. It works well enough, but can cause problems if you add any extra dependencies in that file. Open to alternatives for sharing the code though, perhaps there is a cleaner way.